jit/jit-rules-x86-64.ins

```diff
@@ -3271,6 +3271,19 @@ JIT_OP_MEMCPY: ternary
 	}
 
 JIT_OP_MEMSET: ternary
+	[any, any, imm, if("$3 <= 0")] -> { }
+	[reg, imm, imm, scratch xreg,
+		if("$2 == 0 && $3 <= _JIT_MAX_MEMSET_INLINE && $3 % 16 == 0")] -> {
+		inst = small_block_set(gen, inst, $1, 0, $2, $3, 0, $4, 0, 1);
+	}
+	[reg, imm, imm, scratch reg,
+		if("$3 <= _JIT_MAX_MEMSET_INLINE && $3 < 32")] -> {
+		inst = small_block_set(gen, inst, $1, 0, $2, $3, $4, 0, 0, 0);
+	}
+	[reg, imm, imm, scratch reg, scratch xreg,
+		if("$3 <= _JIT_MAX_MEMSET_INLINE")] -> {
+		inst = small_block_set(gen, inst, $1, 0, $2, $3, $4, $5, 0, 1);
+	}
 	[reg("rdi"), reg("rsi"), reg("rdx"), clobber(creg), clobber(xreg)] -> {
```

Comment:
Please remove these extra empty lines.

Reply:
which extra lines do you mean?

Reply:
in  jit/jit-rules-x86-64.ins starting from 3274